x-pack/plugins/apm/public/components/app/TransactionDetails/Distribution/index.tsx

```diff
@@ -122,12 +137,10 @@ export function TransactionDistribution(props: Props) {
   ]);
 
   // no data in response
-  if (!distribution || distribution.noHits) {
-    // only show loading state if there is no data - else show stale data until new data has loaded
-    if (isLoading) {
-      return <LoadingStatePrompt />;
-    }
-
+  if (
+    (!distribution || distribution.noHits) &&
```

Comment:
Is `noHits` still used?

Reply:
I will fix that while doing this issue. https://github.com/elastic/kibana/issues/82953